src/main/resources/userEmailTemplate-feedbackSessionClosed.html

```diff
@@ -0,0 +1,14 @@
+Hello ${userName},
+The following session is now closed. You can still view your submission by going to the link sent earlier,
```

Comment:
There should be space between "Hello ..." and "The following session...", just like how other similar emails are.